BabylonNative.Android/app/src/main/java/com/android/appviewer/AndroidViewAppActivity.java

```diff
@@ -0,0 +1,201 @@
+package com.android.appviewer;
```

Comment:
This is a copy from what I did in Base3D. Same for the view. Can we release it as OS?